src/models/issue.model.ts

```diff
@@ -1,23 +0,0 @@
-import { Schema, model, ObjectId, Types } from 'mongoose';
-
-interface Issue {
-    reportingUserId: ObjectId;
```

Comment:
This is kinda weird, we might need to resolve this conflict with the main branch